contracts/Kit.sol

```diff
@@ -77,7 +77,7 @@ contract Kit is KitBase {
         token.changeController(tokenManager);
 
         app.initialize();
-        tokenManager.initialize(token, true, 0, true);
+        tokenManager.initialize(token, true, 0);
```

Comment:
Ohh nice catch! Totally didn't realize this had the wrong number of arguments to it due to the [removal of `holders`](https://github.com/aragon/aragon-apps/pull/441/files#diff-2a9b6fae0c66f35810c66745c35ed2e3R67) when this was brought up.